services/provider/server/server.go

```diff
@@ -1834,6 +1835,8 @@ func (s *OCSProviderServer) appendStorageClassKubeResources(
 			klog.Warningf("Encountered unsupported provisioner in storage class %s", storageClassName)
 		} else if storageClass == nil {
 			klog.Warningf("The name %s does not points to a builtin or an existing storage class, skipping", storageClassName)
+		} else if storageClass.Labels[ExternalClassLabelKey] == "true" {
```

Comment:
```suggestion
		} else if strconv.ParseBool(storageClass.Labels[ExternalClassLabelKey]) {
```

Reply:
it returns `value of type (bool, error)`